vagrant/tutorial/post-reboot-bootstrap.py

```diff
@@ -66,12 +66,14 @@
 # Install ZFS.
 check_call(['yum', 'install', '-y', 'zfs'])
 
-# Enable docker.
-# We don't need to start it, since when the box is packaged,
+# Install Docker.
+check_call(["curl", "-o", "/tmp/install-docker.sh", "https://get.docker.com/"])
```

Comment:
The Flocker OS package declared a dependency on some Docker package and the OS package manager arranged for that Docker package to get installed when we installed the Flocker OS package.